utils/treesitter/grammar.js

```diff
@@ -475,11 +475,23 @@ module.exports = grammar({
     extend_base_declaration: ($) =>
       seq('extend', 'base', ':', $._expression, ';'),
 
+    destructor_declaration: ($) =>
+      seq(
+        optional(choice('virtual', 'impl')),
+        'destructor',
+        optional($.deduced_params),
+        choice($.block, ';')
+      ),
+
+    mix_declaration: ($) => seq('__mix', $._expression, ';'),
```

Comment:
Mixins are not officially in carbon (hence the `__` before `mix`), so I'd be find dropping this rule and excluding the mixin tests (your choice).

Reply:
it was simple enough to implement. It is nice to highlight all explorer tests correctly.

Same is case for `__match_first`, but that is also used in prelude.

Reply:
Okay, just know that the explorer syntax for mixins is not the planned mixin syntax for Carbon, which is described here: https://github.com/carbon-language/carbon-lang/blob/trunk/proposals/p2760.md#future-work-mixins
You should have some comment on your mixin rules explaining the situation.